### ISSUE TITLE
Cannot pull updated graphql appsync endpoint.

### ISSUE THREAD

**Issue by Fyb3roptik** [aws-amplify/amplify-cli#13026]
### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

16

### Amplify CLI Version

12.2.2

### What operating system are you using?

Mac

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

AppSync was setup manually, as well as cognito. I have added new redirect urls to cognito as well as updated the graphql endpoint in AppSync. I made these changes in the AWS console but amplify pull is not pulling down the correct information. It is pulling down the old information.

### Describe the bug

AppSync was setup manually, as well as cognito. I have added new redirect urls to cognito as well as updated the graphql endpoint in AppSync. I made these changes in the AWS console but amplify pull is not pulling down the correct information. It is pulling down the old information.

I used postgraphile to setup AppSync as well as manually configured cognito from our own cloudformation.

### Expected behavior

Pull down changes I made in the console

### Reproduction steps

1. Manually create these items in the console
2. Amplify pull does pull it down
3. Make changes to one of these
4. Amplify pull no longer syncs

### Before submitting, please confirm:

- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

**Comment by ykethan** [comment]
Hey @Fyb3roptik, thank you for reaching out. Could you let know if auth resource was created using `amplify add auth` or imported using `amplify import auth`? 
As manual changes to Amplify managed resources on AWS console are currently not supported. 
If the resource are imported using [`amplify import auth`](https://docs.amplify.aws/cli/auth/import/), you should be able to update the user pool on the AWS console and run `amplify pull` to import the changes.

additionally, could you let us know how the GraphQL API was imported into the project. was this using `amplify add codegen --apiId`? how was the endpoint updated and was a new API created?

**Comment by Fyb3roptik** [comment]
Yes I used import auth. I will try to redo that and do a pull thanks.

I did add it via codegen yes

**Comment by ykethan** [comment]
Hey @Fyb3roptik, a quick note on the import auth please ensure the sign out URL has been added to the user pool as well as this would ensure the `aws-exports.js` does fill in the oauth attributes(if present).

Thank you for clarifying the information on the import. could you try removing the reference to the GraphQl API by running `amplify remove api` and then rerunning the `amplify add codegen --apiId <new app-id>` to import the API.
Amplify does not manage the lifecycle of an imported resource.

**Comment by Fyb3roptik** [comment]
This did not work. It still is pulling in an old client id. I now have to commit all of these files into my repo for this to even work.

**Comment by Fyb3roptik** [comment]
I dont want to have to use the terminal prompts to do any updates. Is there not a better way for us to update things like redirect urls?

**Comment by Fyb3roptik** [comment]
I have done all of this several times and it does not work. Like I said, anytime I do this is still pulls the old data and never gives me the new data. This has been my experience every time I have used this on a project and it is very frustrating

**Comment by Fyb3roptik** [comment]
We have decided to go a different way...again...I do not have anymore time to waste on this. Sorry
